web/packages/teleport/src/JoinTokens/JoinTokens.tsx

```diff
@@ -147,7 +147,7 @@ export const JoinTokens = () => {
             width="240px"
             onClick={() => setCreatingToken(true)}
           >
-            Create new Token
+            Create New Token
```

Comment:
@avatus can you manually add this change to your backport PR for join tokens?

Reply:
Yeah I'll add it to the backport PR right now